crates/task-impls/src/quorum_proposal.rs

```diff
@@ -586,6 +610,14 @@ impl<TYPES: NodeType, I: NodeImplementation<TYPES>> QuorumProposalTaskState<TYPE
         event_sender: Sender<Arc<HotShotEvent<TYPES>>>,
     ) {
         match event.as_ref() {
+            HotShotEvent::ProposeNow(view, _) => {
+                self.create_dependency_task_if_new(
```

Comment:
Explicit validation is coming to this before the final integration with consensus.